src/test/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervalsIntegrationTest.java

```diff
@@ -108,13 +111,43 @@ public void testNoIntervals() {
 
     }
 
-    private static Stream<File> getScatteredFiles(final int scatterCount, final File outputDir, String extension) {
-        return IntStream.range(0, scatterCount).mapToObj(n -> new File(outputDir, "000" + n + extension));
+    @Test
+    public void testAsInWGSJointCalling() {
+        //these intervals are small and adjacent -- we want to distribute them among multiple interval lists, but the default behavior is to merge
+        final File wgsCallingIntervals = new File("src/test/resources/hg38.handcurated.noCentromeres.noTelomeres.interval_list");
```

Comment:
publicTestDir instead of hardcoding the path would be better probably

Reply:
Good call. I will do that.